service/awsconfig/v7/templates/cloudformation/guest/instance.go

```diff
@@ -21,6 +21,7 @@ const Instance = `{{define "instance"}}
     DependsOn:
     - MasterInstance
     Properties:
+      Encrypted: true
```

Comment:
For my interest how is the KMS key chosen if its not using `KmsKeyId`?

Reply:
It uses aws/ebs key for region. I checked that in the UI.